packages/happo-target-firefox/src/HappoRunner.js

```diff
@@ -1,4 +1,4 @@
-import { SCREENSHOT_BOX_ID } from './server/Constants';
+import Constants from 'happo-core/lib/Constants';
```

Comment:
needed to import this from `/lib/Constants` because the module overall has some node-only modules like `fs` and `path` that we don't want to include in the browser.